include/trie.h

```diff
@@ -0,0 +1,107 @@
+/*
+    A trie data structure
+*/
+
+#ifndef INCLUDE_TRIE_H_
+#define INCLUDE_TRIE_H_
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+#include "parser.h"
+
+typedef struct trie_t trie_t;
+struct trie_t {
+    char current; 
+        // The first trie_t will be '/0' for any Trie.
```

Comment:
Inline comments should be indented to the same level as the thing they reference, and they should be *above* the thing they're referencing. Please fix all occurrences of this problem.

Reply:
Fixed.